packages/esm-patient-programs-app/src/programs/programs-actions-menu.component.tsx

```diff
@@ -0,0 +1,42 @@
+import { useTranslation } from 'react-i18next';
+import { showModal, useLayoutType } from '@openmrs/esm-framework';
+import { Layer, OverflowMenu, OverflowMenuItem } from '@carbon/react';
+import React, { useCallback } from 'react';
+import { launchPatientWorkspace } from '@openmrs/esm-patient-common-lib';
+
+interface ProgramActionsProps {
+  patientUuid: string;
+  programEnrollmentId: string;
+}
+
+export const ProgramsActionsMenu = ({ patientUuid, programEnrollmentId }: ProgramActionsProps) => {
+  const { t } = useTranslation();
+  const isTablet = useLayoutType() === 'tablet';
+
+  const launchEditProgramsForm = useCallback(
+    () => launchPatientWorkspace('programs-form-workspace', { programEnrollmentId }),
+    [programEnrollmentId],
+  );
+
+  const launchDeleteProgramDialog = () => {
+    const dispose = showModal('program-delete-confirmation-modal', {
+      closeDeleteModal: () => dispose(),
+      programEnrollmentId,
+      patientUuid,
+    });
+  };
+
+  return (
+    <Layer>
```

Comment:
```suggestion
    <Layer  className={styles.layer}>
```

And then the associated styles would look something like this:

```scss
.layer {
  height: 100%;

  :global(.cds--overflow-menu) {
    min-height: unset;
  }
}

.menuItem {
  max-width: none;
}
```

This will give you UI visual consistency with other flows like the Conditions edit experience.